### PR TITLE
HackerSM64 2.0.5

### PR DESCRIPTION
**Pull request by gheskett**

### CODE REVIEW

File: src/audio/heap.c [hunk 1]
```diff
@@ -976,9 +976,6 @@ s32 audio_shut_down_and_reset_step(void) {
 void wait_for_audio_frames(s32 frames) {
     // VC emulator stubs this function because busy loops are not supported
     // Technically we can put infinite loop that _looks_ like -O0 for emu but this is cleaner
-    //if (gIsVC) {
-        return;
```
Comment (arthurtilly) [inline src/audio/heap.c:980]: why is this removed, is this still confirmed to work with VC

Comment (JoshDuMan) [reply src/audio/heap.c:980]: This wouldn't cause issues with VC, it would cause issues (theoretically) for console. But I believe it just causes lag.

Comment (gheskett) [reply src/audio/heap.c:980]: This function goes unused anyway. The reason we haven't deleted it to begin with is because supposedly VC might be able to still find it? Idk how that part works.

Comment (aglab2) [reply src/audio/heap.c:980]: VC can only match strictly binary functions so it just does nothing atm and can be removed.

Comment (arthurtilly) [reply src/audio/heap.c:980]: whats the reason to remove just this line and nothing else then if its unused

Comment (gheskett) [reply src/audio/heap.c:980]: To make it perfectly match vanilla, no other reason. But given what aglab said, we should probably just take it out.

**Review by arthurtilly — APPROVED** [review]
HURRY UP

File: src/audio/heap.c (hunk 1 above)
Comment (Reonu) [reply src/audio/heap.c:980]: so what happened with this

**Review by Reonu — APPROVED** [review]
sure
